cime_config/config_grids.xml

```diff
@@ -3136,6 +3178,14 @@
       <map name="OCN2ATM_SMAPNAME">cpl/gridmaps/SOwISC12to60E2r4/map_SOwISC12to60E2r4-nomask_to_ne30pg2_mono.210119.nc</map>
     </gridmap>
 
+    <gridmap atm_grid="ne30np4.pg2" ocn_grid="ECwISC30to60E2r1">
+      <map name="ATM2OCN_FMAPNAME">cpl/gridmaps/ne30pg2/map_ne30pg2_to_ECwISC30to60E2r1_mono.201006.nc</map>
+      <map name="ATM2OCN_VMAPNAME">cpl/gridmaps/ne30pg2/map_ne30pg2_to_ECwISC30to60E2r1_mono.201006.nc</map>
```

Comment:
Should this be:
```suggestion
      <map name="ATM2OCN_VMAPNAME">cpl/gridmaps/ne30pg2/map_ne30pg2_to_ECwISC30to60E2r1_bilin.201006.nc</map>
```
This seems to be the case for both `EC30to60E2r2` and `SOwISC12to60E2r4`.

Reply:
See this discussion on this meeting page:
https://acme-climate.atlassian.net/wiki/spaces/ECG/pages/2350776353/2-17-2021+Cryosphere+Simulation+Campaign

I think the same applies to all meshes.  Atmospheric velocities should not be interpolated with `mono`/`aave`/`conserve`, but rather with `bilin` or even better something higher order once that's a standard option (e.g. with `mbtempest`).

Reply:
The other `ATM2OCN_VMAPNAME` mapping files are `patch`.  I'm not sure what that is, but it's also used for other meshes so I guess it must be non-conservative?

Reply:
Thanks for catching this @xylar - you're right this certainly should be changed. I've made this new `bilin` mapping file and made the change.